apps/communications/contacts/elements/details.html

```diff
@@ -1,7 +1,7 @@
 <element name="view-contact-details" extends="section">
   <template>
     <gaia-header id='details-view-header' action="back">
-      <h1 id='contact-name-title'></h1>
+      <h1 id='contact-name-title'><bdi class="ellipsis-dir-fix"></bdi></h1>
```

Comment:
you could probably fix the issue by setting `[dir="auto"]` on the `<h1>` or setting `unicode-bidi: isolate` on the CSS